bftengine/include/bcstatetransfer/SimpleBCStateTransfer.hpp

```diff
@@ -125,6 +125,7 @@ struct Config {
   uint32_t maxAcceptableMsgDelayMilli = 60000;                    // 1 minute
   uint32_t sourceReplicaReplacementTimeoutMilli = 5000;           // 5 seconds
   uint32_t fetchRetransmissionTimeoutMilli = 250;                 // ms
+  uint64_t metricsDumpIntervalSeconds = 600;                      // sec
```

Comment:
Can we make this a `std::chrono::duration` instead? We should start to get in the habit of that. I realize that using an int fits as is here, but we really should be using time safe wrappers as much as possible rather than encoding that information in variable names. 

Note that using standard time types will also prevent all the type manipulation code you do further down such as `getMonotonicTimeMilli` and the duration casts.

Reply:
Done